app/services/highlighter/index.ts

```diff
@@ -291,14 +419,17 @@ class HighligherViews extends ViewHandler<IHighligherState> {
 }
 
 @InitAfter('StreamingService')
-export class HighlighterService extends StatefulService<IHighligherState> {
-  static initialState: IHighligherState = {
+export class HighlighterService extends PersistentStatefulService<IHighligherState> {
+  static defaultState: IHighligherState = {
```

Comment:
im not sure i understand why this service state is being made persistent?

Reply:
n the old highlighter all highlight clips were gone after restarting slobs. We wanted to keep the clips in slobs until the user deletes the files or the clips directly in desktop.
I checked a few other services and it looked like using the PersistentStatefulService would be the right thing to do. Is that not the case and if no, what would be the correct way?